server/server.js

```diff
@@ -0,0 +1,5 @@
+let http = require('http')
+let port=process.env.PORT || 3000
+let app = require('./app')
+let server=http.createServer(app)
+server.listen(3000)
```

Comment:
Could you please add a second parameter of a call-back function that console logs the port your server is running on. Currently, someone can not know whether the server is on or not.